docs/index.bs

```diff
@@ -191,6 +191,8 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     A [=/service worker=] has an associated <dfn>start status</dfn> which can be null or a [=Completion=]. It is initially null.
 
+    A [=/service worker=] has an associated <dfn>has had non-no-op fetch listeners flag</dfn>. It is initially true.
```

Comment:
Maybe rename this to reflect the uncertain nature?

```suggestion
    A [=/service worker=] has an associated <dfn>may have had non-empty fetch listener flag</dfn>. It is initially unset.
```

Reply:
I have renamed the flag without negation.  Please let me know what you think.

Reply:
Please feel free to reopen if this still matters.